src/smc-webapp/collaborators/add-to-project.cjsx

```diff
@@ -88,9 +88,9 @@ exports.AddCollaborators = rclass
         replyto      = redux.getStore('account').get_email_address()
         replyto_name = redux.getStore('account').get_fullname()
         if replyto_name?
-            subject = "#{replyto_name} invites you to project #{@props.project.get('title')}"
+            subject = "#{replyto_name} added you to CoCalc project #{@props.project.get('title')}"
```

Comment:
Technically you shouldn't explicitly use "CoCalc" is messages to users.  There's a variable somewhere that gets set.  People can make their own install and call it something else...